nuget/nunitlite/nunitlite.snupkg.nuspec

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata minClientVersion="3.5">
+    <id>NUnitLite</id>
+    <title>NUnitLite</title>
+    <version>$version$</version>
+    <packageTypes>
+      <packageType name="SymbolsPackage" />
+    </packageTypes>
+    <authors>Charlie Poole, Rob Prouse</authors>
+    <owners>Charlie Poole, Rob Prouse</owners>
+    <license type="file">LICENSE.txt</license>
+    <projectUrl>https://nunit.org</projectUrl>
+    <repository type="git" url="https://github.com/nunit/nunit"/>
+    <iconUrl>https://cdn.rawgit.com/nunit/resources/master/images/icon/nunit_256.png</iconUrl>
+    <icon>icon.png</icon>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Symbols for debugging into the NUnitLite assemblies.</description>
```

Comment:
Should we also add a `<language>en-US</language>` for consistency (as we have in nuget/framework/nunit.snupkg.nuspec)?

Reply:
I don't think it matters, but added.